src/termdict/mod.rs

```diff
@@ -74,6 +74,7 @@ const CURRENT_TYPE: DictionaryType = DictionaryType::SSTable;
 
 // TODO in the future this should become an enum of supported dictionaries
 /// A TermDictionary wrapping either an FST based dictionary or a SSTable based one.
+#[cfg_attr(feature = "quickwit", derive(Clone))]
```

Comment:
i needed this to be able to clone the dictionary we pass something valid for 'static to the worker thread, though i don't really like that. The alternative would be to add an Arc inside InvertedIndexReader

Reply:
The fst dictionary cannot be made cloneable?
I prefer either the Arc solution then.

Reply:
> The fst dictionary cannot be made cloneable?

it contains an fst Map. I don't see an obvious reason why that can't be Clone, but it currently isn't, and would require updating tantivy_fst. We could Arc that Map instead or arcing inside the InvertedIndexReader, and make the Dictionary be clone independently of feature flags